internal/persistence/queries.go

```diff
@@ -10,7 +10,7 @@ import (
 
 const (
 	TaskNumLimit    = 300
-	ContentMaxBytes = 3000
+	ContentMaxBytes = 10000
```

Comment:
Maybe a comment about this value that seems arbitrary

Reply:
Thanks for the feedback :)